app/models/user.rb

```diff
@@ -0,0 +1,9 @@
+class User < ApplicationRecord
+  has_many :posts, dependent: :destroy
+  has_many :likes, dependent: :destroy
+  has_many :comments, dependent: :destroy
```

Comment:
- [ ] Kindly add `foreign_key: 'author_id'` to each of the associations.

Reply:
* done 👍